src/modules/Game.class.js

```diff
@@ -50,17 +158,97 @@ class Game {
    * `win` - the game is won;
    * `lose` - the game is lost
    */
-  getStatus() {}
+  getStatus() {
+    return this.#status;
+  }
 
   /**
    * Starts the game.
    */
-  start() {}
+  start() {
+    this.#addNumber();
+    this.#addNumber();
+    this.#status = 'playing';
+  }
 
   /**
    * Resets the game.
    */
-  restart() {}
+  restart() {
+    this.#applyInitialState();
+    this.#score = 0;
+    this.#status = 'idle';
+  }
+
+  #applyInitialState() {
+    this.#state = this.#initialState.map((row) => [...row]);
```

Comment:
The '#applyInitialState' method is duplicating the logic found in the constructor for copying the initial state. Consider creating a utility method for deep cloning arrays to avoid code duplication.